themes/app/src/scss/components/_accommodation.scss

```diff
@@ -0,0 +1,121 @@
+.accommodation__desc {
+    color: black;
+    text-decoration: underline;
+}
+
+.accommodation__arrow {
+    padding-bottom: 0;
+    padding-top: 0;
+    width: 30px;
+    transform: rotate(90deg);
+}
+
+.accommodation__desc h3 {
```

Comment:
<img width="164" alt="screen shot 2018-12-06 at 11 13 47 am" src="https://user-images.githubusercontent.com/27745093/49547624-1334f780-f948-11e8-9400-affaeb4cf80f.png">

Notice how the text is bleeding out of the container? We can apply `display: inline-block` to this class to ensure that our anchor link keeps all its children elements `inline-block` as it is `inline` by default.